fala/settings/base.py

```diff
@@ -184,7 +202,13 @@ def root(*x):
 # Python dotted path to the WSGI application used by Django's runserver.
 WSGI_APPLICATION = "fala.wsgi.application"
 
-INSTALLED_APPS = ("django.contrib.sessions", "django.contrib.messages", "django.contrib.staticfiles", "requests")
+INSTALLED_APPS = (
```

Comment:
@starswan I think this can be in square brackets

Reply:
I think its more idiomatic for this to be a tuple - lists are mutable, this is not.